otwcode/otwarchive
# AO3-4030 Reversi: Lower half of wrangling dash has black-on-black

## Issue

https://otwarchive.atlassian.net/browse/AO3-4030

## Purpose

changes .`dashboard span` to white so it's readable

## Testing Instructions

log in as a tag wrangler, set your site skin to reversi, and check that the dashboard text is white
